viewer/packages/cad-geometry-loaders/src/sector/culling/TakenSectorTree.test.ts

```diff
@@ -5,27 +5,33 @@
 import * as THREE from 'three';
 import { generateSectorTree, expectContainsSectorsWithLevelOfDetail, Mutable } from '../../../../../test-utilities';
 import { PropType } from '../../utilities/reflection';
-import { SectorMetadata, CadModelMetadata, LevelOfDetail } from '@reveal/cad-parsers';
+import {
+  SectorMetadata,
+  CadModelMetadata,
+  LevelOfDetail,
+  BaseSectorMetadata,
+  V8SectorMetadata
+} from '@reveal/cad-parsers';
 import { traverseDepthFirst } from '@reveal/utilities';
 import { DetermineSectorCostDelegate, PrioritizedWantedSector } from './types';
 import { TakenSectorTree } from './TakenSectorTree';
 
-type FacesFile = PropType<SectorMetadata, 'facesFile'>;
+type FacesFile = PropType<BaseSectorMetadata & V8SectorMetadata, 'facesFile'>;
 
 describe('TakenSectorTree', () => {
   const model: CadModelMetadata = {} as any;
   const determineSectorCost: DetermineSectorCostDelegate = () => ({ downloadSize: 1, drawCalls: 1, renderCost: 1 }); // Flat cost
 
   test('default tree contains root as simple', () => {
-    const root = generateSectorTree(2);
+    const root = generateSectorTree(2) as BaseSectorMetadata & V8SectorMetadata;
```

Comment:
Rename `generateSectorTree` to `generateV8SectorTree` and change return type (and fix all usages to not use casting).